server/backend/users/tests/test_serializers.py

```
@@ -0,0 +1,34 @@
+import pytest
+from rest_framework.exceptions import ValidationError
```

Ruff (0.8.2)

`rest_framework.exceptions.ValidationError` imported but unused

Remove unused import: `rest_framework.exceptions.ValidationError`

(F401)